config/locales/server.en.yml

```diff
@@ -6,3 +6,7 @@ en:
     retort_emojis_per_row: "Select how many custom emojis will appear per row (Leave blank to have them all appear in a single row)"
     retort_allow_multiple_reactions: "Allow users to react multiple times to the same post"
     retort_disabled_categories: "List out the categories you'd like to DISABLE retort for, separated by the '|' character"
+    retort_max_per_day: "Maximum number of retorts per user per day."
+    retort_tl2_max_per_day_multiplier: "Increase limit of retorts per day for tl2 (member) by multiplying with this number"
+    retort_tl3_max_per_day_multiplier: "Increase limit of retorts per day for tl3 (member) by multiplying with this number"
```

Comment:
Should this be (member), (trusted), (leader) or whatever the discourse terminology for these levels is?

Reply:
This is the same format used for the like tl limits

Reply:
I mean this:
![captura de pantalla 2018-11-23 a la s 6 30 29 p m](https://user-images.githubusercontent.com/750477/48929464-e5b27c00-ef4d-11e8-9fd2-64ec9dfd1bf7.png)

Should name the trust levels properly